schema/commonmeta/commonmeta.schema.json

```diff
@@ -0,0 +1,541 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "https://commonmeta.org/commonmeta_v0.9.1.json",
+    "title": "Commonmeta v0.9.1",
+    "description": "JSON representation of the Commonmeta schema.",
+    "additionalProperties": false,
+    "definitions": {
+        "affiliations": {
+            "type": "array",
+            "items": {
+                "type": "object",
+                "properties": {
+                    "id": {
+                        "type": "string",
+                        "format": "uri"
+                    },
+                    "name": {
+                        "type": "string"
+                    }
+                },
+                "required": [
+                    "name"
+                ]
+            },
+            "uniqueItems": true
+        },
+        "latitude": {
+            "type": "number",
+            "minimum": -90,
+            "maximum": 90
+        },
+        "longitude": {
+            "type": "number",
+            "minimum": -180,
+            "maximum": 180
+        },
+        "geoLocationPoint": {
+            "type": "object",
+            "properties": {
+                "pointLongitude": {
+                    "$ref": "#/definitions/longitude"
+                },
+                "pointLatitude": {
+                    "$ref": "#/definitions/latitude"
+                }
+            },
+            "required": [
+                "pointLongitude",
+                "pointLatitude"
+            ]
+        }
+    },
+    "type": "object",
+    "properties": {
+        "id": {
+            "description": "The unique identifier for the resource.",
+            "type": "string",
+            "format": "uri",
+            "pattern": "^https?://"
+        },
+        "type": {
+            "description": "The type of the resource.",
+            "enum": [
+                "Article",
+                "Audiovisual",
+                "BookChapter",
+                "BookSeries",
+                "Book",
+                "Component",
+                "Dataset",
+                "Dissertation",
+                "Document",
+                "Grant",
+                "Instrument",
+                "JournalArticle",
+                "JournalIssue",
+                "JournalVolume",
+                "Journal",
+                "PeerReview",
+                "PhysicalObject",
+                "ProceedingsArticle",
+                "ProceedingsSeries",
+                "Proceedings",
+                "ReportComponent",
+                "ReportSeries",
+                "Report",
+                "Software",
+                "Other"
+            ]
+        },
+        "additional_type": {
+            "description": "The additional type of the resource.",
+            "type": "string"
```

Comment:
Do you have any examples of this in use?

Reply:
This comes mainly from DataCite metadata which have `resourceTypeGeneral` and `resourceType`, where the former is from a controlled vocabulary and required and the latter is free text and optional.